### ISSUE TITLE
License bundle normalizer creates duplicate entries in report

### ISSUE THREAD

**Issue by milesgould** [jk1/Gradle-License-Report#50]
With the following in my `build.gradle`:

```
plugins {
    id 'com.github.jk1.dependency-license-report' version '0.6'
}
import com.github.jk1.license.render.*
import com.github.jk1.license.filter.*
licenseReport {
    filters = [ new LicenseBundleNormalizer("config/license-normalizer-bundle.json") ]
    renderer = new InventoryHtmlReportRenderer()
}

[snip]
dependencies {
    compile 'joda-time:joda-time:2.9.7'
[snip]
}
[snip]
```

and the following in `config/license-normalizer-bundle.json`:

```
{
  "bundles" : [
    { "bundleName" : "apache2", "licenseName" : "Apache License, Version 2.0", "licenseUrl" : "http://www.apache.org/licenses/LICENSE-2.0" }
  ],
  "transformationRules" : [
    { "bundleName" : "apache2", "licenseNamePattern" : "Apache 2.*" }
  ]
}
```

I find that `joda-time` appears twice in the generated `index.html`: once under "Apache 2", and once under "Apache License, Version 2.0". I expected it to appear only once, under "Apache License, Version 2.0". The same thing happens if I use `CsvReportRenderer` instead of `InventoryHtmlReportRenderer`.

Is the above adequate to reproduce the problem? If not, I can try to create a more complete reproducer.

**Comment by guenhter** [comment]
Jep. Thx. I'm on that. Have the same problem. 
Reason is, that the license is contained multiple time, but sometimes with two slightly different URL's.
This should of course be merged.

**Comment by guenhter** [comment]
Oh wait, no this is a different problem. This shouldn't happen. I'll have to do some investigations.

**Comment by guenhter** [comment]
The manifest isn't considered for normalisation yes, what causes this problem.
#51 will solve this.

**Comment by guenhter** [comment]
Will be fixed with #56

**Comment by jk1** [comment]
Merged and released in the scope of 0.7
